app/views/advanced_messenger/_issue_notifications.html.erb

```diff
@@ -0,0 +1,106 @@
+<div id="issue_notifications_dialog" >
+    <%=t(:issue_notifications_dialog_text_for_current_user_1)%> <%=User.current.firstname%> <%=User.current.lastname%>, <%=t(:issue_notifications_dialog_text_for_current_user_2)%> <span id="no-unread-notifications"><%=t(:issue_notifications_dialog_text_for_current_user_3)%></span><span id="unread-notifications-count"></span> <span style="color: red"> <%=t(:issue_notifications_dialog_text_for_current_user_4)%> </span> <%=t(:issue_notifications_dialog_text_for_current_user_5)%>.
+    <br/>
+    <a id="go-to-first-one" class="icon icon-down-left-arrow"><%=t(:issue_notifications_dialog_go_to_first_one)%></a> 
+    <a id="status-for-others-link" title="unused but needed by tooltip" class='icon icon-zoom-in' style="color:#169; margin-left:5px"><%=t(:issue_notifications_dialog_status_for_others)%></a>
+    <div id="other-users-status-list-content-holder" style="display:none">
+    </div>
+</div>
+
+<%= javascript_tag do%>
+    function getDialogWidth(screenWidth) {
```

Comment:
Isn't possible to call the function from `application.css` to get the width? Or to get the width with js from sidebar?

I'd like to not replicate computation from redmine that can be changed between versions.

Reply:
Yes. I removed this function and I have taken the width of the sidebar via jquery: 
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/dd5135e4-9557-4b96-9bb5-1ea3665dd461)